modules/ding_reservation/ding_reservation.module

```diff
@@ -67,7 +67,7 @@ function ding_reservation_menu() {
     'access arguments' => array('perform reservation'),
   );
 
-  $items['ding_reservation/%ding_entity/is_reservable'] = array(
+  $items['ding_reservation/%/is_reservable'] = array(
```

Comment:
Found the following tabel (so IE and Edge may have issues):

Browser    |  Address bar   
------------|------------------------------
Chrome      |    32779        
Android      |    8192           
Firefox         |   >64k          
Safari            | >64k           
IE11                | 2047           
Edge 16          |2047

Reply:
Interesting question :)

I have tried to do some calculations:

We have 31 chars in "ding_reservation//is_reservable" and let's say the domain part is 50 chars to be sure. Then we have 1.966 chars left. Each faust is 8 chars and we also need a comma so:

1.966 / 9 = 218,44

So we should be able to handle 218 fausts even in IE11 and Edge. 

This could be a problem on frontpage with a lot carousels. But a lot of them are loaded async at different timings, so there will not be one single request that checks reservability for alle materials on frontpage. Rather it will be split up in 2-3 requests. 
Also considering that the materials in the carousels are lazy loaded on demand themselves I don't think carousels will cause issues with the limitation.

Another case to consider is news/events with material lists. If an administrator adds over 218 materials to a list there could be an issue in IE/Edge, since these will all be checked in one single request.

I guess we could split the request up in max 200 fausts in the client. What do you think?

Reply:
You cannot assume what the length of the local ids is. For example the length of local ids for Icelandic libraries using DDELibra Aleph is 20 chars. For other library systems it might be something else.

You could chunk requests dynamically in the client based on length of local ids and browser but in increases the complexity somewhat. Another approach would be to convert it to a POST request.

If you come up with a method which chucks requests using GET then also remember to set appropriate headers which prevent caching of the response.

Reply:
I think I'll go with the POST request.

But why prevent caching if I was using GET? Reservability is not something that changes often, so I see no problem with the response being cached. Care to elaborate why you think this is necessary?

Reply:
> I do not know what it is for Ereolen

Doesn't matter. FBS know nothing about these ids and only operate on fausts. So it doesn't make sense to look up these ids.

Reply:
Fixed! Using post request instead.

Reply:
> But why prevent caching if I was using GET? Reservability is not something that changes often, so I see no problem with the response being cached. Care to elaborate why you think this is necessary?

You are right. I was mixing up reservability and availability.

Reply:
> As afar as I know Ereolen uses Publizon. Not FBS.

Ah ok. Wasn't aware of that.